regression-tests/backends/gsql-common

```diff
@@ -50,7 +50,7 @@ gsql_master()
         --config-name=$backend --socket-dir=./ --no-shuffle \
         --dnsupdate=yes --resolver=$RESOLVERIP --outgoing-axfr-expand-alias=yes \
         --expand-alias=yes \
-        --cache-ttl=$cachettl --dname-processing \
+        --cache-ttl=$cachettl --domain-cache-ttl=$cachettl --dname-processing \
```

Comment:
I don't think we have Ci with a set cachettl. A fixed value in one of the backends will make sure we test the domain cache in the regressiontests. gsqlite3 is a good candidate (also used for consistent-backends).